scripts/eurostat/birth_death_migration/EurostatNUTS3_BirthDeathMigration.tmcf

```diff
@@ -0,0 +1,49 @@
+
+Node: E:EurostatNUTS3_BirthDeathMigration->E0
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:Count_MortalityEvent
+observationAbout: C:EurostatNUTS3_BirthDeathMigration->geo
+observationDate: C:EurostatNUTS3_BirthDeathMigration->time
+value: C:EurostatNUTS3_BirthDeathMigration->Count_MortalityEvent
+
+Node: E:EurostatNUTS3_BirthDeathMigration->E1
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:Count_BirthEvent_AsAFractionOfCount_Person
```

Comment:
`s/dcs:Count_BirthEvent_AsAFractionOfCount_Person/dcs:Count_BirthEvent_AsAFractionOf_Count_Person/g` (more readable, decided to use this syntax going forward

Reply:
You might have missed this one: AsAFractionOfCount_Person should be AsAFractionOf_Count_Person